libraries/varnish_log.rb

```diff
@@ -102,6 +103,20 @@ def configure_varnish_log
           notifies :restart, "service[#{new_resource.log_format}]", :delayed
         end
 
+        template "#{new_resource.logrotate_path}/#{new_resource.log_format}" do
+          source 'lib_logrotate_varnishlog.erb'
+          path "#{new_resource.logrotate_path}/#{new_resource.log_format}"
```

Comment:
This line might generate a file in `/etc/logrotate.d/varnishlog` or  `/etc/logrotate.d/varnishncsa`. Will it override the default Varnish logrorate provided by the package?

It seems on Ubuntu by example the default file is `/etc/logrotate.d/varnish`